src/routes/admin/export/+server.ts

```diff
@@ -34,7 +34,14 @@ export async function POST({ request }) {
 		title: { type: "UTF8" },
 		created_at: { type: "TIMESTAMP_MILLIS" },
 		updated_at: { type: "TIMESTAMP_MILLIS" },
-		messages: { repeated: true, fields: { from: { type: "UTF8" }, content: { type: "UTF8" } } },
+		messages: {
+			repeated: true,
+			fields: {
+				from: { type: "UTF8" },
+				content: { type: "UTF8" },
+				score: { type: "INT_8", optional: true },
```

Comment:
@lhoestq @severo Out of curiosity, does our viewer support `INT_8` (and optional) data cells in Parquet files?

Reply:
do you have an example file to share?

Reply:
also note that the viewer is not enabled on private datasets !

Reply:
yes i know!